test/FileStorageTest.php

```diff
@@ -21,7 +21,7 @@ public function testDirCreated(): void {
 		$storage = new FileStorage($tmpDir, false);
 
 		$this->assertDirectoryDoesNotExist($dir);
-		$storage->createDir($dir);
+		$storage::createDir($dir);
```

Comment:
Static calls should use ClassName instead of dynamic variable value. But actually, maybe it's just better to change the signature of method to be non static